### PR TITLE
JBTM-3430 Create a quarkus coordinator uber jar and remove JaxRsActiv…

### PR DESCRIPTION
**Pull request by mmusgrov**
https://github.com/mmusgrov/narayana/pull/new/JBTM-3430

!MAIN !CORE !QA_JTA !QA_JTS_JDKORB !QA_JTS_OPENJDKORB !QA_JTS_JACORB !BLACKTIE !XTS !PERF NO_WIN !RTS !AS_TESTS !TOMCAT !JACOCO
LRA

This PR is needed for the lra quickstarts to be able to use the quarkus based coordinator (and I will open a quickstart PR that depends on this later today).
It also removes JaxRsActivator because there can only be one such activator per application.

### CODE REVIEW

File: rts/lra/coordinator-quarkus/pom.xml [hunk 1]
```diff
@@ -152,5 +152,14 @@
               </plugins>
           </build>
       </profile>
+      <profile>
+        <id>quarkus-uber</id>
+        <activation>
+          <activeByDefault>true</activeByDefault>
+        </activation>
+        <properties>
+          <quarkus.package.type>uber-jar</quarkus.package.type>
```
Comment (xstefank) [inline rts/lra/coordinator-quarkus/pom.xml:161]: just curious, did you try if it works with fast-jar?

**Review by xstefank — APPROVED** [review]

File: rts/lra/coordinator-quarkus/pom.xml (hunk 1 above)
Comment (mmusgrov) [reply rts/lra/coordinator-quarkus/pom.xml:161]: I'll give it a go

Comment (mmusgrov) [reply rts/lra/coordinator-quarkus/pom.xml:161]: It's nice but I've decided to go with the standard maven packaging for this one. If someone wants to raise a PR for it then please do but bear in mind that we are pulling in the resulting uber jar from the [quickstart](https://github.com/mmusgrov/quickstart-1/blob/JBTM-3430/rts/lra-examples/lra-coordinator/pom.xml#L63). It wasn't clear how to package up the resulting jar (target/quarkus-app/quarkus-run.jar) into the lra-coordinator-quarkus artefact for consumption by the quickstart.

Comment (xstefank) [reply rts/lra/coordinator-quarkus/pom.xml:161]: sure, we can take this later

**Review by xstefank — APPROVED** [review]

**Comment by mmusgrov** [comment]
Thanks @xstefank

**Pull request merged**